R/slidy_presentation.R

```diff
@@ -31,7 +31,8 @@
 #' render("pres.Rmd", slidy_presentation(incremental = TRUE))
 #' }
 #' @export
-slidy_presentation <- function(incremental = FALSE,
+slidy_presentation <- function(number_sections = FALSE,
```

Comment:
Because pandoc-related argument comes first in other formats (e.g., `html_document`), and `number_sections` here works as if pandoc's option.
For formats with `toc`-related arguments, I placed `number_section` after them.

Reply:
Thanks I did not notice that. This could break position argument call but I don't think anyone is using those format functions without naming the argument, do you ?

Reply:
I don't think so, too, and that's why the current order.